packages/orchestrator/internal/sandbox/block/tracker_test.go

```diff
@@ -0,0 +1,109 @@
+package block
+
+import (
+	"testing"
+)
+
+func TestTracker_AddAndHas(t *testing.T) {
+	const pageSize = 4096
+	tr := NewTracker(pageSize)
+
+	offset := int64(pageSize * 4)
+
+	// Initially should not be marked
+	if tr.Has(offset) {
+		t.Errorf("Expected offset %d not to be marked initially", offset)
+	}
```

Comment:
maybe a little easier to understand?

```suggestion
	hasOffset := tr.Has(offset)
	assert.False(t, hasOffset)
```